docs/page-objects/inputs.rst

```diff
@@ -84,3 +84,121 @@ You may define your own input classes if you are using a :ref:`framework
 
 However, note that custom input classes may make your :ref:`page object classes
 <page-object-classes>` less portable across frameworks.
+
+
+.. _input-validation:
+
+Input validation
```

Comment:
Having it on a sepatate page makes sense to me, because the content look quite different from a general "inputs" page. 

I was thinking about the general "inputs" topic as something about explaining the concept of page object inputs - it's an uncommon concept.

On the other hand, input validation is a specific feature of page objects, with a prominent use case, which I think we should call out earlier: I think the majority of uses of validate_input would be dealing with unstable websites, which randomly return blank or invalid pages.

Reply:
OK, I’ll revert that. But would it be OK to move the topic above additional requests in the table of contents? (i.e. which topic should come first, input validation or additional requests?).

Reply:
I don't have a strong opinion on that (haven't thought about that), but a first reaction is that additional request is a more widely used feature, so it should come before.